shared/chat/conversation/messages/wrapper/wrapper-timestamp/index.js

```diff
@@ -75,23 +66,37 @@ class _WrapperTimestamp extends React.Component<Props & OverlayParentProps, Stat
       }
     }
   }
+  _onMouseOver = () => {
+    this.setState(o => (o.showMenuButton ? null : {showMenuButton: true}))
+  }
   _setShowingPicker = (showingPicker: boolean) =>
     this.setState(s => (s.showingPicker === showingPicker ? null : {showingPicker}))
+
   render() {
     const props = this.props
     return (
       <Box style={styles.container}>
         {props.orangeLineAbove && <Box style={styles.orangeLine} />}
         {!!props.timestamp && <Timestamp timestamp={props.timestamp} />}
         <HoverBox
-          className={props.showingMenu || this.state.showingPicker ? 'active' : ''}
+          className={[
```

Comment:
should this add `props.decorate && 'WrapperTimestamp-decorated'` and remove the selectors in `HoverBox` above?

Reply:
thats in the next pr which is on top of this one. i only pulled down the css cause we needed it here too since we show the menu once its been made

Reply:
oh nvm I see that's what you did in the other PR, LGTM